plugin/src/index.ts

```diff
@@ -3,7 +3,7 @@ import { ConfigPlugin, createRunOncePlugin } from "expo/config-plugins";
 import withAndroidTwilioVideoWebrtc from "./withAndroidTwilioVideoWebrtc";
 import withIosTwilioVideoWebrtc from "./withIosTwilioVideoWebrtc";
 
-const pkg = require("react-native-twilio-video-webrtc/package.json");
+const pkg = require("../../package.json");
```

Comment:
a relative path works for plugin consumers that are using a `symlink` for the library.

Otherwise, this will error when those consumers trying to run `npx expo prebuild`
![Screenshot 2023-06-16 at 3 39 19 PM](https://github.com/blackuy/react-native-twilio-video-webrtc/assets/30332608/fc67baa4-1045-4482-925f-f5d75ac2a6aa)